src/decays/decay.cpp

```diff
@@ -46,6 +46,28 @@ std::size_t hash_pid_list(int pid_in, Container pids_out)
 
 } // anonymous namespace
 
+extern int PDG_id_pairs[PDG_id_pairs::NUMBER_OF_PDG_IDS];
```

Comment:
I think this line is incorrect and must be removed. (We define an enum class in the header with the same name.)